atlas-search/using-atlas-search.mdx

```diff
@@ -0,0 +1,157 @@
+---
+slug: /atlas-search/using-atlas-search
+title: Using Atlas Search
+description: Using Atlas Search to improve your search results.
+---
+
+## Prerequisites
+
+Atlas Search requires the [WPGraphQL](https://wordpress.org/plugins/wp-graphql/) plugin.
+
+## Querying with Atlas Search
+
+Once your site is set up with an Atlas Search license and the initial content sync has completed, Atlas Search will take over the default WordPress search functionality and enhance your WordPress search experience.
+
+The WPGraphQL plugin is a good place to test search queries. WPGraphQL provides a ‘GraphiQL IDE' in the WordPress dashboard where you can interact with the WPGraphQL Schema in the browser. Atlas Search overrides the default search functionality for WPGraphQL queries on all Atlas Search supported WordPress data types.
+
+Below is a screenshot of the WPGraphQL plugin & GraphiQL IDE view, where Atlas Search has overridden the default WP search behavior, followed by some sample queries:
+
+![Atlas Search Plugin Data Sync](./images/as-wpgraphiql.png)
```

Comment:
The text in this image is really small, could you bump up text size for the screenshot?